src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age !== undefined && (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      )}
+
+      {person.isMarried === true ? (
+        <p className="Person__partner">{`${person.partnerName} is my ${personPartnerType}`}</p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
```suggestion
      <p className="Person__partner">
         {person.isMarried ?`${person.partnerName} is my ${personPartnerType}` : 'I am not married'}
      </p>
```
redundant to compare boolean with boolean, better to make ternary inside element (just text will be changed)